frontend_lib/src/component/AppContent/AppContentRightMenu.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { ROLE, APP_FEATURE_MODE } from '../../helper.js'
+import EmojiReactionButtons from '../../container/EmojiReactionButtons.jsx'
+import SelectStatus from '../Input/SelectStatus/SelectStatus.jsx'
+import ArchiveDeleteContent from '../OptionComponent/ArchiveDeleteContent.jsx'
+// require('./AppContentRightMenu.styl') // see https://github.com/tracim/tracim/issues/1156
+
+const AppContentRightMenu = (props) => (
+  <div className='appContentRightMenu'>
+    {props.loggedUser.userRoleIdInWorkspace >= ROLE.contributor.id && (
+      <>
+        <EmojiReactionButtons
```

Comment:
```suggestion
        <EmojiReactionButtonList
```
maybe?

Reply:
Mhm, `EmojiReactionButtons` does not really return a list. I tried to do what was done for breadcrumbs, but I would not mind EmojiReactionButtonList. @Skylsmoi any opinion?

Reply:
We went for EmojiReactions with @Skylsmoi